BrainPortal/app/models/data_provider.rb

```diff
@@ -635,6 +635,7 @@ def cache_collection_index(userfile, directory = :all, allowed_types = :regular)
       if userfile.is_a? FileCollection
         if directory == :all
           entries = Dir.glob(userfile.name + "/**/*")
+          #entries = Dir.glob(userfile.name + "/**/*", File::FNM_DOTMATCH).reject { |p| p.ends_with?("/.") || p.ends_with?("/..")}
```

Comment:
Any reason to have this line commented ?

Reply:
While I was testing all this I discovered a behavior of CBRAIN I might want to change in the future. But I am not sure. And that line is how the behavior would change.